globe_search.py

```diff
@@ -0,0 +1,40 @@
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.chrome.service import Service as ChromeService
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.webdriver.chrome.options import Options
+import time
+
+def searcher_name_games(game: str) -> list:
+    game_html_format = game.replace(' ', '+')
+    search_game = game_html_format.lower()
+    apostrophe = ["'", "®"]
+    for symbol in apostrophe:
+        search_game = search_game.replace(symbol, '')
+
+    # Setup Selenium WebDriver
+    options = Options()
+    options.headless = True  # Run in headless mode
+    driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()), options=options)
+
+    url = f'https://www.playstation.com/en-us/search/?q={search_game}&category=games'
+    driver.get(url)
+    
+    # Wait for the page to load
+    time.sleep(5)
```

Comment:
Static delays are very bad, is there any way to wait for element?

Reply:
remove for asyncio